test/svm/SvmSpoke.Fill.ts

```diff
@@ -302,7 +307,7 @@ describe("svm_spoke.fill", () => {
     await setCurrentTime(program, state, relayer, new BN(relayData.fillDeadline + 1));
 
     // Close the fill PDA
-    await program.methods.closeFillPda(relayHash, relayData).accounts(closeFillPdaAccounts).signers([relayer]).rpc();
+    await program.methods.closeFillPda().accounts(closeFillPdaAccounts).signers([relayer]).rpc();
 
     // Verify the fill PDA is closed
```

Comment:
could you add a test to show that closing the fill PDA works if the fill was done with across plus alternative structure?

Reply:
sure, will add that